src/components/dashboard/components/SideNavigation.jsx

```diff
@@ -19,16 +18,23 @@ class SideNavigation extends React.Component {
       <div className="side-nav d-none d-md-block d-lg-block">
         <ul className="nav flex-column">
           <li className="nav-item">
-            <a className="nav-link active" href="#" onClick={this.handleShow}>
-              Add Dictionary
-            </a>
-            <a href="/dashboard" className="nav-link active" >
-              Sources
-            </a>
-            <a href="/dashboard/concepts" className="nav-link active">
+        <a className="nav-link active" onClick={this.handleShow}>
+          Add Dictionary
+        </a>
+        <a href="/dashboard/concepts" className="nav-link active">
               <i className="fa fa-tags" /> Concepts
-            </a>
+          </a>
           </li>
+        <li className="nav-item">
```

Comment:
why did you put dictionaries in their own <li></li> why not follow the convention

Reply:
because it is a list item